pkgs/by-name/gs/gsad/package.nix

```
@@ -0,0 +1,126 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gnutls
+, gnutar
+, libgcrypt
+, cmake
+, glib
+, libxml2
+, libxslt
+, pkg-config
+, gettext
+, doxygen
+, gvm-libs
+, libtasn1
+, libidn2
+, p11-kit
+, python3Packages
+, pcre2
+, installShellFiles
+, util-linux
+, libselinux
+, libsepol
+, git
+, libmicrohttpd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gsa";
+  version = "23.0.0";
+
+  src1 = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gsa";
+    rev = "v${version}";
+    hash = "sha256-yA+B6fh8C07tGEoy/+FPvcvx3D1OfPig0Eoqv2oJfsA=";
+  };
+
+  src2 = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gsad";
+    rev = "v22.9.1";
+    hash = "sha256-FTB+YqsAi4EgCGMSbWmDLjh2NHzpyA9zpPMAzm90Thg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    doxygen
+    pkg-config
+  ];
+
+  buildInputs = [
+    libxslt
+    git
+    gettext
+    glib
+    pcre2
+    libgcrypt
+    gnutls
+    gnutar
+    gvm-libs
+    util-linux
+    installShellFiles
+    libmicrohttpd
+    libidn2
+    p11-kit
+    libxml2
+    libselinux
+    libsepol
+    python3Packages.polib
+    libtasn1
+  ];
+
+  unpackPhase = ''
+    mkdir -p $out/share/gvm/gsad/web
+    mkdir -p $out/gsad
+    cp -R $src1/* $out/share/gvm/gsad/web
+    cp -R $src2/* $out/gsad
```

```suggestion
    cp -R $src2/* $out/gsad
```

Why this makes sense? Two sources? Two versions? You need to explain this. Add comments.